earth_enterprise/legacy_installer/README.txt

```diff
@@ -0,0 +1 @@
+This is the obsolete installer. See ../src/installer for the scripts you want.
```

Comment:
@andreisel would it be better to move that script (any anything else that is still being used) into `src/installer` ?